### PR TITLE
[PT-BR] noPropertyAccessFromIndexSignature.md

### PR DESCRIPTION
**Pull request by jvitormf**
Translation from en to pt
@khaosdoctor e @danilofuchs

### CODE REVIEW

File: docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Esta configuração garante a consistência entre acessar o campo pela sintaxe "ponto" (`obj.chave`), e "indexado" (`obj["chave"]`) e a forma que a propriedade é declarada no tipo.
+
+Sem esta flag, o Typescript irá permitir que você use a sintaxe de ponto para acessar os campos que não estão definidos:
+
+```ts twoslash
+// @errors: 4111
+declare function obterConfiguracoes(): configuracoesDoJogo;
+// ---cut---
+interface configuracoesDoJogo {
```
Comment (danilofuchs) [inline docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md:14]:
```suggestion
interface ConfiguracoesDoJogo {
```

Interfaces should always be PascalCase

File: docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+//       ^?
+```
+
+Ativar esta flag irá gerar um erro porque o campo desconhecido usa a sintaxe ponto invés da sintaxe indexada.
+
+```ts twoslash
+// @errors: 4111
+// @noPropertyAccessFromIndexSignature
+declare function obterConfiguracoes(): configuracoesDoJogo;
+interface configuracoesDoJogo {
```
Comment (danilofuchs) [inline docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md:42]:
```suggestion
interface ConfiguracoesDoJogo {
```

**Review by danilofuchs — CHANGES_REQUESTED** [review]
Nice one!
Just a small concertn

File: docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md (hunk 1 above)
Comment (jvitormf) [reply docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md]: @danilofuchs obrigado pela dica

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
LGTM

**Comment by khaosdoctor** [comment]
LGTM

**Comment by khaosdoctor** [comment]
@orta I think this might be an error

**Comment by orta** [comment]
I wonder if it's a bad offset - this fail is real:

Screen Shot 2021-02-25 at 9 25 44 PM

File: docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+oneline: "Força o uso de acessadores indexados para chaves declaradas usando um tipo indexado"
+---
+
+Esta configuração garante a consistência entre acessar o campo pela sintaxe "ponto" (`obj.chave`), e "indexado" (`obj["chave"]`) e a forma que a propriedade é declarada no tipo.
+
+Sem esta flag, o Typescript irá permitir que você use a sintaxe de ponto para acessar os campos que não estão definidos:
+
+```ts twoslash
+// @errors: 4111
+declare function obterConfiguracoes(): configuracoesDoJogo;
```
Comment (danilofuchs) [inline docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md:12]:
```suggestion
declare function obterConfiguracoes(): ConfiguracoesDoJogo;
```

File: docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+configuracoes.nomeDeUsuario;
+//       ^?
+```
+
+Ativar esta flag irá gerar um erro porque o campo desconhecido usa a sintaxe ponto invés da sintaxe indexada.
+
+```ts twoslash
+// @errors: 4111
+// @noPropertyAccessFromIndexSignature
+declare function obterConfiguracoes(): configuracoesDoJogo;
```
Comment (danilofuchs) [inline docs/tsconfig/pt/options/noPropertyAccessFromIndexSignature.md:41]:
```suggestion
declare function obterConfiguracoes(): ConfiguracoesDoJogo;
```

**Review by danilofuchs — COMMENTED** [review]
CI is failing on these two lines

**Comment by jvitormf** [comment]
I'm sorry. This is my first contribution to an open-source project, I'll pay more attention next time.
I already committed the changes suggested
Thank you all

**Comment by danilofuchs** [comment]
You did a great job @jvitormf!. Thank you for the help!

LGTM

**Pull request merged**